pep-0363.txt

```diff
@@ -5,247 +5,256 @@ Last-Modified: $Date$
-              are calls of a variable named "getattr" whose default
-              value is the builtin "getattr";
+    c.5   uses which would have to stay as "getattr" because they
+          are calls of a variable named "getattr" whose default
+          value is the builtin "getattr";
 
-        c.5   uses of the 2-argument form, inside a try/except block
-              which catches AttributeError and uses a default value
-              instead; these could use 2-argument form of the new
-              syntax;
+    c.5   uses of the 2-argument form, inside a try/except block
+          which catches AttributeError and uses a default value
+          instead; these could use 2-argument form of the new
+          syntax;
 
-        c.10  uses of "delattr", which could use the new syntax.
+    c.10  uses of "delattr", which could use the new syntax.
 
-    As examples, the line
+As examples, the line::
 
-        setattr(self, attr, change_root(self.root, getattr(self, attr)))
+    setattr(self, attr, change_root(self.root, getattr(self, attr)))
 
-    from Lib/distutils/command/install.py could be rewritten
+from Lib/distutils/command/install.py could be rewritten::
 
-        self.(attr) = change_root(self.root, self.(attr))
+    self.(attr) = change_root(self.root, self.(attr))
 
-    and the line
+and the line::
 
-        setattr(self, method_name, getattr(self.metadata, method_name))
+    setattr(self, method_name, getattr(self.metadata, method_name))
 
-    from Lib/distutils/dist.py could be rewritten
+from Lib/distutils/dist.py could be rewritten::
 
-        self.(method_name) = self.metadata.(method_name)
+    self.(method_name) = self.metadata.(method_name)
 
 
 Performance Impact
+==================
 
-    Initial pystone measurements are inconclusive, but suggest there may
-    be a performance penalty of around 1% in the pystones score with the
-    patched version.  One suggestion is that this is because the longer
-    main loop in ceval.c hurts the cache behaviour, but this has not
-    been confirmed.
+Initial pystone measurements are inconclusive, but suggest there may
+be a performance penalty of around 1% in the pystones score with the
+patched version.  One suggestion is that this is because the longer
+main loop in ceval.c hurts the cache behaviour, but this has not
+been confirmed.
 
-    On the other hand, measurements suggest a speed-up of around 40--45%
-    for dynamic attribute access.
+On the other hand, measurements suggest a speed-up of around 40--45%
+for dynamic attribute access.
 
 
 Error Cases
+===========
 
-    Only strings are permitted as attribute names, so for instance the
-    following error is produced:
+Only strings are permitted as attribute names, so for instance the
+following error is produced::
 
-     >>> x.(99) = 8
-        Traceback (most recent call last):
-          File "<stdin>", line 1, in <module>
-        TypeError: attribute name must be string, not 'int'
+ >>> x.(99) = 8
```

Comment:
@brettcannon Yes, this renders properly. Because there is one space at the beginning of line, I will change it to four spaces.